chaco/lineplot.py

```diff
@@ -449,14 +449,6 @@ def _line_width_changed(self):
         self.invalidate_draw()
         self.request_redraw()
 
-    def __getstate__(self):
-        state = super(LinePlot, self).__getstate__()
-        for key in ["traits_view"]:
```

Comment:
Is `traits_view` a transient trait? I dont see anything in the code or in the documentation saying that it is.

Reply:
I was going off wha Corran said in this issue description: "This is removing the traits_view trait from the state - under normal Traits, this will never be in the state because of special handling of Views, so this whole method can be removed."
This should certainly be documented though